src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureAnalyzer.cs

```diff
@@ -137,6 +137,31 @@ public abstract class DetectPreviewFeatureAnalyzer : DiagnosticAnalyzer
             return null;
         }
 
+        protected virtual SyntaxNode? GetConstraintSyntaxNodeForTypeConstrainedByPreviewTypes(ISymbol typeOrMethodSymbol, ISymbol previewInterfaceConstraintSymbol)
+        {
+            return null;
+        }
+
+        protected virtual SyntaxNode? GetPreviewReturnTypeSyntaxNodeForMethodOrProperty(ISymbol methodOrPropertySymbol, ISymbol previewReturnTypeSymbol)
+        {
+            return null;
+        }
+
+        protected virtual SyntaxNode? GetPreviewParameterSyntaxNodeForMethod(IMethodSymbol methodSymbol, ISymbol parameterSymbol)
+        {
+            return null;
+        }
+
+        protected virtual SyntaxNode? GetPreviewTypeArgumentSyntaxNodeForMethod(IMethodSymbol methodSymbol, ISymbol parameterSymbol)
+        {
+            return null;
+        }
```

Comment:
I don't see where this is called?

Reply:
I forgot to push the last commit that deleted this method. Fixed now :)